stdlib/optparse.pyi

```diff
@@ -53,7 +53,6 @@ class HelpFormatter:
     short_first: Any
     width: int
     def __init__(self, indent_increment: int, max_help_position: int, width: Optional[int], short_first: int) -> None: ...
-    def _format__Text(self, _Text: _Text) -> _Text: ...
```

Comment:
There is a `_format_text` method, I guess this is the result of a bad search-replace. Seems fine to remove it though since it's private.